connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsIntegrationBaseTest.java

```diff
@@ -234,6 +245,8 @@ public void startClusters(Map<String, String> additionalMM2Config) throws Except
     @AfterEach
     public void shutdownClusters() throws Exception {
         try {
+            primaryProducer.close();
+            backupProducer.close();
```

Comment:
We probably still want to try to gracefully shut down the cluster even if we encounter an error with stopping our producers:
```suggestion
            Utils.closeQuietly(primaryProducer);
            Utils.closeQuietly(backupProducer);
```